manifests/charts/UPDATING-CHARTS.md

```diff
@@ -55,7 +55,6 @@ This will regenerate the Go structs used for schema validation.
 ## Step 4. Update the generated manifests
 
 Tests of istioctl use the auto-generated manifests to ensure that the istioctl binary has the correct version of the charts.
-These manifests can be found in [gen-istio.yaml](../charts/istio-control/istio-discovery/files/gen-istio.yaml).
 To regenerate the manifests, run:
```

Comment:
Why did you remove it?

Reply:
the file has been removed(https://github.com/istio/istio/blob/a04ccfde0f984dd0b17a4298acdddb30c0e850f8/releasenotes/notes/drop-kustomize.yaml#L8),   or has it been moved somewhere?

Reply:
You are right.